draft-gpew-priv-ppm.md

```diff
@@ -493,6 +493,9 @@ of the aggregators is configured with following parameters:
   specified. See issue#161.]
 * `agg_auth_key`: A key for HMAC-SHA256 {{?RFC2104}} shared by the aggregators and
   used for message authentication during the aggregation flow.
+* `collect_auth_key`: A key for HMAC-SHA256 {{?RFC2104}} shared by the
+  aggregators and the collector and used for message authentication during the
+  collect flow.
```

Comment:
This key would only need to be shared by the collector and leader. The helper doesn't need this key, does it?